protostar/configuration_file/configuration_file_v1_test.py

```diff
@@ -37,7 +37,7 @@ def protostar_toml_path_fixture(protostar_toml_content: str, project_root_path:
 @pytest.fixture(name="configuration_file")
 def configuration_file_fixture(protostar_toml_path: Path, project_root_path: Path):
     return ConfigurationFileV1(
-        ConfigurationTOMLReader(path=protostar_toml_path),
+        ConfigurationTOMLReader(path=protostar_toml_path, ignore_attribute_casing=True),
```

Comment:
FYI I think this sounds more like an ideal candidate for composition/inheritance 😉 

`CaseInsensitiveKeysAdapter(ConfigurationTOMLReader(path))`

Reply:
I created a new `ConfigurationTOMLReader` so this is not necessary anymore.